tools/keycloak/sync/Pims.Tools.Keycloak.Sync.csproj

```diff
@@ -37,7 +37,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.9" />
     <PackageReference Include="Microsoft.Extensions.Options.DataAnnotations" Version="3.1.9" />
     <PackageReference Include="System.CommandLine.DragonFruit" Version="0.3.0-alpha.20070.2" />
-    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.8.0" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="6.11.0" />
```

Comment:
I thought these were already completed.  I don't understand why your branch is showing that it will be changed again?